Tests/pens/roundingPen_test.py

```diff
@@ -0,0 +1,67 @@
+from fontTools.misc.fixedTools import floatToFixedToFloat
+from fontTools.pens.recordingPen import RecordingPen, RecordingPointPen
+from fontTools.pens.roundingPen import RoundingPen, RoundingPointPen
+from functools import partial
+
+
+tt_scale_round = partial(floatToFixedToFloat, precisionBits=14)
```

Comment:
the test rightly fails. floatToFixedToFloat simply does `otRound(value * scale) / scale` (with `scale = 1 << precisionBits`), so it comes out with a lot of decimal positions. It's meant to give you a float that is quantized to the fractional portions of the F2Dot14, to do computations with it. Instead you want to use `floatToFixedToStr` here (and convert the str back to float()) so you obtain a normalized, short representation suitable for pretty strings, like the hashpointpen

Reply:
> Does pytest compare string representations, or is that just done for the diff output?

no, pytest compares the actual values just using the built-in `assert`; the computed and expected values _are_ different!

Reply:
you were misled into thinking that the Transform object actually contain `0.913025` because its `repr()` prints that string rounded to 6 decimal positions, but it _actually_ contains a longer float

Reply:
> Instead you want to use `floatToFixedToStr` here (and convert the str back to float()) so you obtain a normalized, short representation suitable for pretty strings, like the hashpointpen

That's why I originally suggested that ;).

Reply:
my fault, thanks behdad